test/rekt/features/broker/feature.go

```diff
@@ -200,3 +203,208 @@ func brokerRedeliveryDropN(retryNum int32, dropNum uint) *feature.Feature {
 
 	return f
 }
+
+func BrokerDeadLetterSinkExtensions() *feature.FeatureSet {
+	fs := &feature.FeatureSet{
+		Name: "Knative Broker - DeadLetterSink - with Extensions",
+
+		Features: []*feature.Feature{
+			brokerSubscriberUnreachable(),
+			brokerSubscriberErrorNodata(),
+			brokerSubscriberErrorWithdata(),
+		},
+	}
+	return fs
+}
+
+func brokerSubscriberUnreachable() *feature.Feature {
+	f := feature.NewFeatureNamed("Broker Subscriber Unreachable")
+
+	source := feature.MakeRandomK8sName("source")
+	sink := feature.MakeRandomK8sName("sink")
+	via := feature.MakeRandomK8sName("via")
+
+	eventSource := "source1"
+	eventType := "type1"
+	eventBody := `{"msg":"test msg"}`
+	event := cloudevents.NewEvent()
+	event.SetID(uuid.New().String())
+	event.SetType(eventType)
+	event.SetSource(eventSource)
+	event.SetData(cloudevents.ApplicationJSON, []byte(eventBody))
+
+	//Install the broker
+	brokerName := feature.MakeRandomK8sName("broker")
+	f.Setup("install broker", broker.Install(brokerName, broker.WithEnvConfig()...))
+	f.Requirement("broker is ready", broker.IsReady(brokerName))
+	f.Requirement("broker is addressable", broker.IsAddressable(brokerName))
+
+	f.Setup("install sink", eventshub.Install(sink, eventshub.StartReceiver))
+
+	// Point the Trigger subscriber to the sink svc.
+	cfg := []manifest.CfgFn{
+		trigger.WithSubscriber(nil, "http://fake.svc.cluster.local"),
+		trigger.WithDeadLetterSink(svc.AsKReference(sink), ""),
+	}
+
+	// Install the trigger
+	f.Setup("install trigger", trigger.Install(via, brokerName, cfg...))
```

Comment:
see here, regarding the `fake...` url:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/test/e2e_new/dls_extensions_test.go#L88-L93